README.md

```diff
@@ -1,8 +1,17 @@
 # starknet.kt
 StarkNet SDK for Kotlin
 
+## Development
 
-## Ensuring idiomatic Java code
+### Hooks
```

Comment:
I don't think these are hooks. I'd name this section `Running tests and lint` or something similiar.

Reply:
Those are hooks.